packages/@aws-cdk/toolkit/lib/actions/deploy/index.ts

```diff
@@ -182,7 +184,7 @@ export interface DeployOptions extends BaseDeployOptions {
   /**
    * What kind of security changes require approval
    *
-   * @default RequireApproval.Broadening
+   * @default RequireApproval.NEVER
```

Comment:
i think this should be the default in programmatic land? not that it matters right now since requireApproval is TODO